elements/lisk-transactions/src/transfer.ts

```diff
@@ -33,9 +33,16 @@ export interface TransferInputs {
 	readonly nonce: string;
 	readonly networkIdentifier: string;
 	readonly data?: string;
-	readonly passphrase?: string;
 	readonly recipientId?: string;
 	readonly recipientPublicKey?: string;
+	readonly senderPublicKey?: string;
+	readonly passphrase?: string;
+	readonly passphrases?: ReadonlyArray<string>;
+	readonly keys?: {
+		readonly mandatoryKeys: Array<Readonly<string>>;
+		readonly optionalKeys: Array<Readonly<string>>;
+		readonly numberOfSignatures: number;
```

Comment:
same here

Reply:
Since this was updated recently on your PR I need to check the impact on this one.

Reply:
Removed from sign() interface, functions and tests